integration_tests/src/main/python/window_function_test.py

```diff
@@ -455,3 +456,55 @@ def test_window_aggs_for_rows_collect_list():
             (partition by a order by b,c_int rows between CURRENT ROW and UNBOUNDED FOLLOWING) as collect_struct
         from window_collect_table
         ''')
+
+
+# Generates some repeated values to test the deduplication of GpuCollectSet.
+# And GpuCollectSet does not yet support struct type.
+_gen_data_for_collect_set = [
+    ('a', RepeatSeqGen(LongGen(), length=20)),
+    ('b', IntegerGen()),
+    ('c_int', RepeatSeqGen(IntegerGen(), length=15)),
+    ('c_long', RepeatSeqGen(LongGen(), length=15)),
+    ('c_time', RepeatSeqGen(DateGen(), length=15)),
+    ('c_string', RepeatSeqGen(StringGen(), length=15)),
+    ('c_float', RepeatSeqGen(FloatGen(), length=15)),
+    ('c_decimal', RepeatSeqGen(DecimalGen(precision=8, scale=3), length=15)),
```

Comment:
Could you add in a few more types here?  We say we support bytes, shorts, doubles, and dates. It would be really nice to add them to the tests too.

Reply:
Done.